recipes/publish.rb

```diff
@@ -50,45 +50,17 @@
   #########################################################################
 
   # update a data bag with the artifact build info
-  # TODO: (jtimberman) This is not the first time this has been used in
-  # a delivery build cookbook. It's probably time to create a Chef
-  # resource for this functionality
-  ruby_block 'track-artifact-data' do # ~FC014
-    block do
-      load_delivery_chef_config
-      proj = Chef::DataBag.new
-      proj.name(project_name)
-      proj.save
+  load_delivery_chef_config
+  chef_data_bag project_name
```

Comment:
Line 53 is [from](https://github.com/chef-cookbooks/delivery-sugar/blob/264081a73e613be7e9d12f0b8b290e374884b795/libraries/delivery_dsl.rb#L381)`delivery-sugar`.

Line 54 is in [cheffish](https://github.com/chef/cheffish/blob/master/lib/chef/resource/chef_data_bag.rb)